src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -472,9 +472,15 @@ trait ChoosesMode {
         case Some(WALK) =>
           responsePlaceholders = makeResponsePlaceholders(withRouting = true)
           makeRequestWith(withTransit = true, Vector(bodyStreetVehicle))
+        case Some(EMERGENCY) =>
+          responsePlaceholders = makeResponsePlaceholders(withRouting = true)
+          makeRequestWith(withTransit = true, Vector(bodyStreetVehicle))
         case Some(WALK_TRANSIT) =>
           responsePlaceholders = makeResponsePlaceholders(withRouting = true)
           makeRequestWith(withTransit = true, Vector(bodyStreetVehicle))
+        case Some(EMERGENCY_TRANSIT) =>
+          responsePlaceholders = makeResponsePlaceholders(withRouting = true)
+          makeRequestWith(withTransit = true, Vector(bodyStreetVehicle))
```

Comment:
These 4 cases contains the same code. Could we use a single case `Some(WALK | WALK_TRANSIT | EMERGENCY .. )` here ?